ingest/downloader/downloader.py

```diff
@@ -23,13 +24,28 @@ def create_workbook(self, input_json: dict) -> Workbook:
         for ws_title, ws_elements in input_json.items():
             if ws_title == 'Project':
                 worksheet: Worksheet = workbook.create_sheet(title=ws_title, index=0)
+            elif ws_title == SCHEMAS_WORKSHEET:
+                continue
             else:
                 worksheet: Worksheet = workbook.create_sheet(title=ws_title)
 
             self.add_worksheet_content(worksheet, ws_elements)
 
+        self.generate_schemas_worksheet(input_json, workbook)
+
         return workbook
 
+    def generate_schemas_worksheet(self, input_json, workbook):
+        schemas = input_json.get(SCHEMAS_WORKSHEET)
+        if not schemas:
+            raise Exception('The schema urls are missing')
+        schemas_worksheet = workbook.create_sheet(SCHEMAS_WORKSHEET)
+        schemas_worksheet.cell(row=1, column=1, value=SCHEMAS_WORKSHEET)
+        row_num = 1
+        for schema in schemas:
+            row_num += 1
```

Comment:
Did you consider using `enumerate()`

```suggestion
        for row_num, schema in enumerate(schemas, start=1):
```